application/templates/secret.yaml

```diff
@@ -15,9 +15,21 @@ metadata:
   annotations:
 {{ toYaml $.Values.secret.annotations | indent 4 }}
 {{- end }}
+{{- if .data }}
 data:
 {{- range $key, $value := .data }}
   {{ $key }}: {{ $value | b64enc }}
 {{- end }}
+{{- else if .stringData }}
+stringData:
+{{- range $key, $value := .stringData }}
+  {{ $key }}: {{ $value }}
+{{- end }}
+{{- else if .encodedData }}
+data:
+{{- range $key, $value := .encodedData }}
+  {{ $key }}: {{ $value }}
+{{- end }}
+{{- end }}
```

Comment:
This feels like a similar case, where there is no problem with providing this functionality (adding base64-encoded content) alongside existing ones (plain text).

Could you come up with a way to provide Base64-encoded strings alongside plain text to create a `Secret` through this chart?

Reply:
Done!